src/public_service_finder/templates/admin_only.html

```diff
@@ -2,46 +2,60 @@
     <div class="container mx-auto px-4 py-8 max-w-[1400px] min-w-[1000px]">
-        <!-- Header section -->
         <div class="flex flex-col items-center mb-8 mt-16">
-            <h1 class="text-4xl font-bold text-gray-100">Admin Dashboard</h1>
+            <h1 class="text-4xl font-bold text-gray-900 dark:text-gray-100">Admin Dashboard</h1>
         </div>
 
-        <!-- Pending Services Section -->
-        {% if pending_services %}
-        <div class="overflow-x-auto bg-gray-800 rounded-xl shadow-sm p-8">
+        {% if services_with_changes %}
+        <div class="overflow-x-auto bg-white dark:bg-gray-800 rounded-xl shadow-sm p-8">
             <table class="min-w-full text-base">
                 <thead>
-                    <tr class="bg-gray-700">
-                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-300 uppercase tracking-wider">Service Name</th>
-                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-300 uppercase tracking-wider">Description</th>
-                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-300 uppercase tracking-wider">Address</th>
-                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-300 uppercase tracking-wider">Status</th>
-                        <th class="px-8 py-4 text-center text-sm font-bold text-gray-300 uppercase tracking-wider">Actions</th>
+                    <tr class="bg-gray-100 dark:bg-gray-700">
+                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Service Name</th>
+                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Description</th>
+                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Address</th>
+                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Status</th>
+                        <th class="px-8 py-4 text-left text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Changes</th>
+                        <th class="px-8 py-4 text-center text-sm font-bold text-gray-700 dark:text-gray-300 uppercase">Actions</th>
                     </tr>
                 </thead>
-                <tbody class="bg-gray-800">
-                    {% for service in pending_services %}
-                    <tr id="service-row-{{ service.id }}" class="hover:bg-gray-700 transition-colors duration-200">
-                        <td class="px-8 py-6 whitespace-nowrap text-base font-medium text-gray-100">{{ service.name }}</td>
-                        <td class="px-8 py-6 text-base text-gray-300">{{ service.description }}</td>
-                        <td class="px-8 py-6 text-base text-gray-300">{{ service.address }}</td>
+                <tbody class="bg-white dark:bg-gray-800 divide-y divide-gray-200 dark:divide-gray-700">
```

Comment:
![WhatsApp Image 2024-12-03 at 21 28 07_37777ea5](https://github.com/user-attachments/assets/57d7b262-b439-45f9-b7a3-3235f69ca13c)
Maybe this line is leading to the white background when no entries are there? @deepjyotk